srv/modules/runners/validate.py

```diff
@@ -283,6 +283,30 @@ def mgrs(self):
         else:
             self.passed['mgrs'] = "valid"
 
+    def formats(self):
+	"""
+	As ceph-disk does indirectly not support two types of disk formats
+	we intervene as early as possible
+        See https://github.com/SUSE/DeepSea/issues/436
+	"""
+        formats = {}
+        for node in self.data.keys():
+            # we can detect disk-formats only in the new yaml format
+            if ('ceph' in self.data[node] and
+                'storage' in self.data[node]['ceph']):
+                devices = self.data[node]['ceph']['storage']['osds']
+                formats[node] = []
+                for dev_name in devices.keys():
+                    if 'format' in devices[dev_name]:
```

Comment:
if ```format``` is not a key in the profile _I think_ we default to bluestore. Maybe we should add an else branch here?